### PR TITLE
Document new way to indicate cross-posts

### LINKED ISSUE #223: document new way to indicate cross-posts
in YAML as opposed to text https://github.com/ropensci/roweb3/pull/652

### PR DESCRIPTION
**Pull request by maelle**
Fix #223

### CODE REVIEW

**Review by yabellini — APPROVED** [review]
Looks good. Thanks!

File: editor.Rmd [hunk 1]
```diff
@@ -73,6 +73,21 @@ blogguidance::show_checklist(c("editor-checklist.csv", "editor-checklist-peer-re
...
+```yaml
+crossposts:
+- name: website1 name
+  url: url to post on website1
+- name: website2 name
+  url: url to post on website2
+```
+
+This is to make the cross-post alert appear ([example](https://ropensci.org/blog/2023/08/01/r-universe-and-cran/)
+as well as to make the cross-post metadata appear in the JSON feed that Rogue Scholar consumes.
```
Comment (steffilazerte) [inline editor.Rmd:89]:
```suggestion
This is to ensure the cross-post alert appears ([example](https://ropensci.org/blog/2023/08/01/r-universe-and-cran/)
as well as to make the cross-post metadata visible to the JSON feed that Rogue Scholar uses.
```

**Comment by steffilazerte** [comment]
Very cool! Just one small comment. I also submitted a [couple](https://github.com/ropensci/roweb3/issues/671) of [issues](https://github.com/ropensci/roweb3/issues/670) on roweb3 regarding formatting and some potentially missed blog posts. Thanks @maelle!

**Comment by maelle** [comment]
Thank you both!!

**Pull request merged**